### PR TITLE
feat(chart): added-choice-of-log-format-json-for-deployment-driver

### PR DESCRIPTION
**Pull request by IshinMV**
Which issue(s) this PR fixes:
Issue https://github.com/longhorn/longhorn/issues/10064

What this PR does / why we need it:
Added support for JSON log format configuration in Longhorn-driver

Special notes for your reviewer:
Additional documentation or context

### CODE REVIEW

File: chart/templates/deployment-driver.yaml [hunk 1]
```diff
@@ -25,6 +25,9 @@ spec:
           command:
           - longhorn-manager
           - -d
+          {{- if eq .Values.longhornManager.log.format "json" }}
```
Comment (COLDTURNIP) [inline chart/templates/deployment-driver.yaml:28]: Do you mean `longhornDriver.log.format` ?

```suggestion
          {{- if eq .Values.longhornDriver.log.format "json" }}
```

Comment (IshinMV) [reply chart/templates/deployment-driver.yaml]: Yes, sorry, i fixed it

**Comment by IshinMV** [comment]
@derekbit I'm sorry, but
it is necessary to repeat the forced movement of the Feature/master/log-format-json-for-deployment-driver branch from c96ec30 to 82e311f 
because DCO is fixed and commit added https://github.com/longhorn/longhorn/pull/10072/commits/c4ca2effd5ebb81c49a025be35077cae73febddd

**Review by COLDTURNIP — APPROVED** [review]
LGTM

**Review by derekbit — APPROVED** [review]

**Comment by derekbit** [comment]
@IshinMV Thanks for your contribution.

**Comment by derekbit** [comment]
@mergify backport v1.8.x

**Comment by IshinMV** [comment]
@derekbit You may to add backport v1.7.x and v1.6.x too?

**Comment by derekbit** [comment]
@mergify backport v1.7.x v1.6.x

**Pull request merged**